src/views/Timeline.vue

```diff
@@ -134,7 +134,7 @@
 
 		<NcModal v-if="showAlbumPicker"
 			key="albumPicker"
-			:label-id="t('photos', 'album-picker')"
+			:label-id="t('photos', 'album picker')"
```

Comment:
Should this even be translated? I think not.
```suggestion
			label-id="album-picker"
```

Reply:
It is a technical property of the component, so it is not rendered.